src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.tsx

```diff
@@ -1,39 +1,22 @@
-import PropTypes from "prop-types";
-import React from "react";
+import * as React from "react";
 
-import Button from "../../inline/Button";
+import Button, { ButtonProps } from "../../inline/Button";
```

Comment:
Will remove the properties if needed when I do my components.

Reply:
Did you mean "add" not "remove" in your sentence? You should not remove the props that I exported here, haha.